packages/tldraw/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './Tldraw'
 export * from './types'
 export * from './state/shapes'
+export * as shapeStyleHelpers from './state/shapes/shared/shape-styles'
```

Comment:
This feels like a strange one off to have as a top level export.

Can we try adding `export * from './shared` to ./state/shapes/index.ts` instead?

https://github.com/gamma-app/tldraw/blob/main/packages/tldraw/src/state/shapes/index.ts